src/blockchain_txn.proto

```diff
@@ -56,6 +57,7 @@ message blockchain_txn {
         blockchain_txn_payment_v2 payment_v2 = 24;
         blockchain_txn_price_oracle_v1 price_oracle_submission = 25;
         blockchain_txn_gen_price_oracle_v1 gen_price_oracle = 26;
+        blockchain_txn_transfer_hotspot_v1 transfer_hotspot = 27;
```

Comment:
```suggestion
        blockchain_txn_transfer_hotspot_v1 transfer_hotspot_v1 = 27;
```
?

Reply:
We usually don't version `v1`, but `v2` or later gets one. At least that's been the pattern so far.

Reply:
Makes sense!